src/amo/components/Page/index.js

```diff
@@ -31,10 +36,28 @@ type InternalProps = {|
 export const PageBase = ({
   children,
   clientApp,
+  errorHandler,
   isHomePage = false,
   location,
   showWrongPlatformWarning = true,
 }: InternalProps) => {
+  let errorContent;
+  if (errorHandler && errorHandler.hasError()) {
+    log.warn(`Captured API Error: ${errorHandler.capturedError.messages}`);
```

Comment:
I'm aware this is just refactoring, but do we need all 4** errors to be logged as warning level?  It must happen _a lot_?

Reply:
Good point. I can downgrade it to `debug`.

Reply:
Or maybe I should make it `info`.